src/main/io/gps.h

```diff
@@ -65,13 +65,19 @@ typedef enum {
     GPS_AUTOBAUD_ON
 } gpsAutoBaud_e;
 
+typedef enum {
+    GPS_USE_GALILEO_OFF = 0,
+    GPS_USE_GALILEO_ON
+} gpsUseGalileo_e;
+
 #define GPS_BAUDRATE_MAX GPS_BAUDRATE_9600
 
 typedef struct gpsConfig_s {
     gpsProvider_e provider;
     sbasMode_e sbasMode;
     gpsAutoConfig_e autoConfig;
     gpsAutoBaud_e autoBaud;
+    gpsUseGalileo_e useGalileo;
```

Comment:
Can you make this an `uint8_t` here please? Using explicitly sized types is cleanest for structs that are used in parameter groups, and enums actually use 32 bits which is a lot of overhead.

Also, I would probably ditch the `gpsUseGalileo_e` enum, and treat this as a bool. Creating enums that essentially reinvent bools doesn't make much sense.

Reply:
It was strange to me too, but I did it like the others GPS params.

Reply:
It's a bad pattern, and it should be fixed at some point, so there is no point in adding more repeats of the bad pattern.